src/server/job_recov_db.c

```diff
@@ -825,4 +804,51 @@ job_or_resv_recov_db(char *id, int objtype)
 		return (job_recov_db(id));
 	}
 }
+
+/**
+ * @brief
+ *		Print backtrace of the program
+ *
+ * @see
+ * 		job_save_db
+ *
+ * @param[in]	jobid - Print call trace of the job
+ *
+ */
+void
+print_backtrace(char *jobid) {
```

Comment:
Would be good to move the implementation to an utility library. I wont mandate it but leave it to you and other reviewers/ maintainers

Reply:
@Shrini-h , I think for the time being it's good to be here only and however it's restricted to server only.